app/data/models.py

```diff
@@ -34,7 +35,45 @@ class ActionTypes(object):
             (UPDATE, 'Update'),
             (DELETE, 'Delete')
         )
+
         @classmethod
         def as_list(cls):
             return [cls.CREATE, cls.UPDATE, cls.DELETE]
+
     action = models.CharField(max_length=6, choices=ActionTypes.choices)
+
+
+class CeleryJob(models.Model):
```

Comment:
agreed, makes sense to get rid of the task type if this is the only likely use case.  Worst case we expand it later